index.html

```diff
@@ -555,30 +555,65 @@ <h2>
         mode</a>.
       </p>
       <p>
-        Each <a>display mode</a>, except <code>browser</code>, has a
-        <dfn>fallback display mode</dfn>, which is the <a>display mode</a> that
-        the user agent can try to use if it doesn't support a particular
-        <a>display mode</a>. If the user agent does support a <a>fallback
-        display mode</a>, then it checks to see if it can use that <a>display
-        mode</a>'s <a>fallback display mode</a>. This creates a fallback chain,
-        with the <a>default display mode</a> (<code>browser</code>) being the
-        last item in the chain.
+        The <dfn>fallback chain</dfn> for:
+      </p>
+      <ol>
+        <li>{{browser}} is «{{browser}}».
```

Comment:
Optional: I think I would prefer if the fallback chain didn't include itself, even if it complicates the algorithm. Otherwise, it's theoretically possible for a display mode to not include itself, which would be weird.

(On the other hand, it allows for "virtual" display modes that don't introduce a new mode, but represent a preset ordered list of other modes. It seems unlikely we'd want this however.)

Reply:
done :)